src/betterproto/lib/google/protobuf/__init__.py

```diff
@@ -2,10 +2,13 @@
 # sources: google/protobuf/any.proto, google/protobuf/api.proto, google/protobuf/descriptor.proto, google/protobuf/duration.proto, google/protobuf/empty.proto, google/protobuf/field_mask.proto, google/protobuf/source_context.proto, google/protobuf/struct.proto, google/protobuf/timestamp.proto, google/protobuf/type.proto, google/protobuf/wrappers.proto
 # plugin: python-betterproto
 import warnings
-from dataclasses import dataclass
+
 from typing import Dict, List
 
+from dataclasses import dataclass
+
 import betterproto
+
 from betterproto.grpc.grpclib_server import ServiceBase
```

Comment:
Which setting is putting from and normal imports on separate lines, this looks weird?

Reply:
Iirc that's pep8 recommendation. 

From the pep

```py
import mypkg.sibling
from mypkg import sibling
from mypkg.sibling import example
```

Reply:
Sorry I meant with the new line inbetween

Reply:
`lines_between_types`. This just looks weird here becomes i think its a single import and then `from` import from the same package.